docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -157,6 +157,8 @@ Note: When joining multiple networks you should use the **--network name:mac=\<m
 
 The *host* network mode should be configured in the YAML file.
```

Comment:
@Luap99 Didn't you change this bit?

Reply:
Yes, this line should be removed. Looks like I missed that.